### PR TITLE
ID-363 Reworked the brand colour pallet according to the new accessible rules

### PR DESCRIPTION
**Pull request by WarwickGoatee**
image

### CODE REVIEW

**Comment by WarwickGoatee** [comment]
Confirmed with Sara that she's happy with this.

**Comment by lol768** [comment]
We're confident that making them brighter like this doesn't reduce contrast (where they're used as a background) then, I take it?

**Comment by matmannion** [comment]
Can we include a sample of the secondary colour as well? Ideally with the text on top for both so we can see what the suggested text colour is

**Comment by WarwickGoatee** [comment]
> Can we include a sample of the secondary colour as well? Ideally with the text on top for both so we can see what the suggested text colour is

Could have it so it shows the hexes as background colour with contrast test on top?

**Comment by WarwickGoatee** [comment]
Done it

File: less/variables.less [hunk 1]
```diff
@@ -7,14 +7,14 @@
 @container-large-desktop:   (1150px + @grid-gutter-width);
 
 // Primary colour palette   // Secondary colour palette
-@id7-brand-purple: #5b3069; // hsl(285, 37%, 41%) originally #7b428e now hsl(285, 37%, 30%)
```
Comment (matmannion) [inline less/variables.less:10]: These comments are presumably all wrong now

**Review by matmannion — APPROVED** [review]

**Pull request merged**